src/data/scene.ts

```diff
@@ -35,19 +55,9 @@ export const SCENE_SAVED_ATTRIBUTES = {
     "media_content_id",
     "media_content_type",
   ],
-  climate: [
-    "target_temperature",
-    "target_temperature_high",
-    "target_temperature_low",
-    "target_humidity",
-    "fan_mode",
-    "swing_mode",
-    "hvac_mode",
-    "preset_mode",
-  ],
-  vacuum: ["cleaning_mode"],
-  fan: ["speed", "current_direction"],
-  water_heather: ["temperature", "operation_mode"],
+  timer: ["duration"],
+  vacuum: ["cleaning_mode", "fan_speed"],
```

Comment:
I'm not aware of an attribute `cleaning_mode` of vacuum devices. The base vacuum devices only know `fan_speed`.